### PR TITLE
V1.3 delete leave feature

### PR DESCRIPTION
**Pull request by JasonRay168**
Added tests for delete leave command, but also fixed some code issues with equals method of add and delete leave commands

### CODE REVIEW

**Review by LordSaumya — COMMENTED** [review]
Code looks good, but please add tests

File: src/main/java/seedu/address/logic/commands/AddLeaveCommand.java [hunk 1]
```diff
@@ -129,7 +129,9 @@ public boolean equals(Object other) {
         }
 
         AddLeaveCommand otherAddLeaveCommand = (AddLeaveCommand) other;
-        return targetId.equals(otherAddLeaveCommand.targetId);
+        return targetId.equals(otherAddLeaveCommand.targetId)
+                && from.equals(otherAddLeaveCommand.from)
```
Comment (inezkok) [inline src/main/java/seedu/address/logic/commands/AddLeaveCommand.java:133]: good catch of checking the dates too!

File: src/test/java/seedu/address/logic/parser/DeleteLeaveCommandParserTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+        Id id = new Id("EID1234-5678");
+        LocalDate startDate = LocalDate.parse("2023-12-26", DateTimeFormatter.ISO_LOCAL_DATE);
+        LocalDate endDate = LocalDate.parse("2023-12-28", DateTimeFormatter.ISO_LOCAL_DATE);
+
+        DeleteLeaveCommand expectedDeleteLeaveCommand = new DeleteLeaveCommand(id, startDate, endDate);
+
+        assertParseSuccess(parser, userInput, expectedDeleteLeaveCommand);
+    }
+}
```
Comment (inezkok) [inline src/test/java/seedu/address/logic/parser/DeleteLeaveCommandParserTest.java:48]: good testing!

**Review by inezkok — APPROVED** [review]
good job with the testings!

**Pull request merged**